projection.tex

```diff
@@ -74,11 +74,138 @@
 %
 \end{abstract}
 
-\section{Projection Method}
 %
 \label{s:projection}
-
-TBA.
+\section{Govern equations for 2D incompressible flow}
+For simplicity, suppose $\rho=1$. \\
+Momentum equation in x-direction:
+\begin{align*}
```

Comment:
To help reading the plaintext, there should be a blankline starting with a comment mark `%` to distinguish the English and the display equations.